src/internal/di/PlatformModule.browser.ts

```diff
@@ -0,0 +1,14 @@
+import { IdGenerator } from '../IdGenerator'
+import { BrowserIdGenerator } from '../IdGenerator.browser'
+import { PlatformModule } from './PlatformModule'
+
+export class BrowserPlatformModule implements PlatformModule {
```

Comment:
PlatformModule encapsulates platform-specific implementations. For now, we only have IdGenerator.